databases/week1/hyf-week1.sql

```diff
@@ -0,0 +1,51 @@
+USE `hyf-week1`;
+ 
+SELECT * FROM `task`;
+SELECT * FROM `user`;
+SELECT * FROM `status`;
+SELECT `status`. `name` FROM `task`;
+SELECT * FROM `task` WHERE status_id != 3;
+-- Find out how many tasks are in the task table
+SELECT COUNT(id) FROM `task`;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT * FROM `task`WHERE due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT * FROM `task` WHERE status_id = 3;
+
+-- Find all the tasks that are not marked as done
+SELECT * FROM `task` WHERE status_id < 3;
+
+-- Get all the tasks, sorted with the most recently created first
+SELECT * FROM `task` ORDER BY due_date DESC;
+
+-- Get the single most recently created task
+SELECT * FROM `task` ORDER BY due_date DESC LIMIT 1;
```

Comment:
Why order by `due_date` instead of `created `as it is asking to sort out by when a task was created?
It should be something as below
```
SELECT id,title,created
FROM task
ORDER BY created DESC;
``` 
and
```
SELECT id,title,created
FROM task
ORDER BY created DESC LIMIT 1;
```
 respectively